pkg/broker/types.go

```diff
@@ -60,6 +60,7 @@ type ProvisionRequest struct {
 	PlanID            uuid.UUID      `json:"plan_id"`
 	ServiceID         uuid.UUID      `json:"service_id"`
 	SpaceID           uuid.UUID      `json:"space_guid"`
+	Context           apb.Context    `json:"context,omitempty"`
```

Comment:
I'm sure what to do with it at this point before the Context spec is a little more firm.  Some of our APBs default the namespace to a newly created project if absent so when writing this I wanted that to work.  

I think we should just be returning a 400 in the handler if it's not included. Perhaps we should wait until the spec is finalized though and add the check in the handler if it's a "required" field.